README.rst

```diff
@@ -62,65 +176,231 @@ A simple example to get you up and running:
   print("Results:")
   print(result)
 
-Example output:
+You should see the following output printed by Python:
 
-.. code-block::
+.. code-block:: text
 
   Circuit:
   (0, 0): ───X^0.5───M('m')───
   Results:
   m=11000111111011001000
 
+Congratulations! You have run your first quantum simulation in Cirq. You can
+continue learning more by exploring the `many Cirq tutorials
+<#tutorials>`__ described below.
 
-Feature requests / Bugs / Questions
------------------------------------
 
-If you have feature requests or you found a bug, please `file them on GitHub <https://github.com/quantumlib/Cirq/issues/new/choose>`__.
+Cirq Documentation
+------------------
 
-For questions about how to use Cirq post to
-`Quantum Computing Stack Exchange <https://quantumcomputing.stackexchange.com/>`__ with the
-`cirq <https://quantumcomputing.stackexchange.com/questions/tagged/cirq>`__ tag.
+Documentation for Cirq is available in a variety of formats.
 
-How to cite Cirq
-----------------
 
-Cirq is uploaded to Zenodo automatically. Click on the badge below to see all the citation formats for all versions.
+Tutorials
```

Comment:
The original README had a "Tutorials" link pointing to https://quantumai.google/cirq/build which is absent in the new version.  I feel that page is very useful for new users, because the tutorials there are categorized by subject and link to nicely rendered notebooks, no need to figure and run colabs at all.

Can we add a leading bullet point for these?  For example

* Tutorials at the Cirq homepage, for example, circuit building and circuit simulation tutorials under the [Build](https://quantumai.google/cirq/build) and [Simulate](https://quantumai.google/cirq/simula) tabs respectively.  Please other tabs for more tutorials.

Reply:
I think what happened is that it seemed like the /start/basics link would be an easier starting point for new people. We can add this, though – your text provides good info.